kong/db/migrations/operations/200_to_210.lua

```diff
@@ -345,7 +345,9 @@ local cassandra = {
         for _, row in ipairs(rows) do
           local set_list = { "ws_id = " .. default_ws }
           for _, key in ipairs(unique_keys) do
-            table.insert(set_list, render([[$(KEY) = '$(WS):$(VALUE)']], {
+            local template = row[key] and [[$(KEY) = '$(WS):$(VALUE)']] or
+                                          [[$(KEY) = '$(WS):']]
+            table.insert(set_list, render(template, {
```

Comment:
@locao I wonder if we should be setting these to "workspace-colon-empty-string" and if this won't cause trouble down the line with actual empty-string values. What if instead we just _not_ add them to `set_list`, so that null values are not rendered as part of the `UPDATE`? In other words, `if row[key] then table.insert(...` — does that work?

Reply:
Fixed!

Reply:
on second thought... we actually need to make this into a new migration that fixes this, otherwise people who already migrated into 2.1.0 won't get this fixed. :\